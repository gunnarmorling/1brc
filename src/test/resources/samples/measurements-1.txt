Kunming;19.8
