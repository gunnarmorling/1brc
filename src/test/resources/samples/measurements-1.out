{Kunming=19.8/19.8/19.8}
